src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -22,21 +22,23 @@ namespace qmcplusplus
 RotatedSPOs::RotatedSPOs(const std::string& my_name, std::unique_ptr<SPOSet>&& spos)
     : SPOSet(my_name),
       OptimizableObject(my_name),
-      Phi(std::move(spos)),
+      Phi_(std::move(spos)),
       nel_major_(0),
-      params_supplied(false),
+      params_supplied_(false),
       apply_rotation_timer_(createGlobalTimer("RotatedSPOs::apply_rotation", timer_level_fine))
 {
-  OrbitalSetSize = Phi->getOrbitalSetSize();
+  OrbitalSetSize = Phi_->getOrbitalSetSize();
 }
 
 RotatedSPOs::~RotatedSPOs() {}
 
 
 void RotatedSPOs::setRotationParameters(const std::vector<RealType>& param_list)
 {
-  params          = param_list;
-  params_supplied = true;
+  params_.resize(param_list.size());
+  std::transform(param_list.begin(), param_list.end(), params_.begin(), [](RealType val) { return ValueType(val); });
```

Comment:
Just leaving a note that this will need to be converted to a proper unpacking into ValueType in a subsequent PR. Beyond the scope of what is being done here though